scripts/addUnsafeDefaultClient.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+# Part of the SW360 Portal Project.
+#
+# SPDX-License-Identifier: EPL-1.0
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+set -e
+
+AUTH_CLIENTS_URL="http://127.0.0.1:5984/sw360oauthclients"
+TRUSTED_CLIENT_URL="${AUTH_CLIENTS_URL}/trusted-sw360-client"
+
+getDefaultAppRev() {
+    if ! type jq &>/dev/null; then
+        >&2 echo "This script needs the 'jq' tool, to parse JSON"
+        exit 1
+    fi
+
+    curl --silent \
+         -X GET "$TRUSTED_CLIENT_URL" |
+        jq -r '._rev'
+}
+
+createDefaultApp() {
+    curl --silent \
+         --output /dev/null \
+         -X PUT "$TRUSTED_CLIENT_URL" \
+         -d @- <<EOF
+{
+   "_id": "trusted-sw360-client",
+   "authorities": [
+       "BASIC"
+   ],
+   "secretRequired": true,
+   "scoped": true,
+   "description": "my first test client",
+   "scope": [
+       "READ", "WRITE", "ADMIN"
+   ],
+   "client_id": "trusted-sw360-client",
+   "client_secret": "sw360-secret",
+   "access_token_validity": 3600,
+   "refresh_token_validity": 3600,
+   "authorized_grant_types": [
```

Comment:
Will not be used by the server anyway, so remove

Reply:
I did not get during the first review that you are writing directly to the couch and not through the rest interface of the auth server. So you were right, we need these props!